packages/salesforcedx-utils-vscode/src/helpers/validations.ts

```diff
@@ -8,6 +8,7 @@ export function isInteger(value: string | undefined): boolean {
   return (
     value !== undefined &&
     !/\D/.test(value) &&
+    // tslint:disable-next-line:radix
     Number.isSafeInteger(Number.parseInt(value))
```

Comment:
Add a radix param instead of disabling tslint?

Reply:
Some of these files were accidentally changed while we were getting her workspace set up and will be reverted